src/display/editor/stamp.js

```diff
@@ -190,6 +190,9 @@ class StampEditor extends AnnotationEditor {
       this.#canvas = null;
       this.#observer?.disconnect();
       this.#observer = null;
+      if (this.#resizeTimeoutId !== null) {
+        clearTimeout(this.#resizeTimeoutId);
+      }
```

Comment:
For consistency with the rest of the patch:
```suggestion
      if (this.#resizeTimeoutId) {
        clearTimeout(this.#resizeTimeoutId);
        this.#resizeTimeoutId = null;
      }
```